api/db/repository/complaint.py

```diff
@@ -39,8 +41,23 @@ def get_by_id(_db: Session, complaint_id: int):
     return db_complaint
 
 
-def get_all(_db: Session, offset: int = 0, limit: int = 100):
-    return _db.query(Complaint).offset(offset).limit(limit).all()
+def get_all(
+    _db: Session,
+    offset: int = 0,
+    limit: int = 100,
+    statuses: List[str] = None,
+):
+    query = _db.query(Complaint)
+    if statuses:
+        if set(statuses) <= {i.value for i in Status}:
```

Comment:
```suggestion
        if set(statuses) < {i.value for i in Status}:
```
if they are equal this filter is not needed

Reply:
Yes, you right. I fixed it.